.github/workflows/tests.yml

```diff
@@ -46,6 +46,28 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           fail_ci_if_error: false # too many upload errors to keep "true"
+      - name: Make sure the CLI stays fast
+        id: cli-load-time
+        run: |
+          PYTHONPROFILEIMPORTTIME=1 g2p -h 2> importtime.txt > /dev/null
+          CLI_LOAD_TIME="$((/usr/bin/time --format=%E g2p -h > /dev/null) 2>&1)"
+          echo "CLI load time: $CLI_LOAD_TIME"
+          cat importtime.txt
+          echo "CLI load time: $CLI_LOAD_TIME" > import-message.txt
+          echo "PR head ${{ github.event.pull_request.head.sha }}" >> import-message.txt
+          echo "Imports that take more than 0.1 s:" >> import-message.txt
+          grep -E 'cumulative|[0-9]{6}' importtime.txt >> import-message.txt
+          if [[ "$CLI_LOAD_TIME" > "0:01.00" ]]; then \
```

Comment:
wow, does this work?

Reply:
Yeah, it's sweet. We've been using this in EveryVoice, that's where I copied it from.

Reply:
OK, it's a bit clunky, but in CI I don't mind that too much, and the message you get in the PR conversation is really nice.

As for `"$CLI_LOAD_TIME" > "0:01.00"` doing the right thing, I force the time output to that syntax with `/usr/bin/time --format=%E` higher up.